pkgs/development/libraries/hfst/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, autoreconfHook
+, bison
+, flex
+, fetchFromGitHub
+, gettext
+, icu
+, stdenv
+, swig
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hfst";
+  version = "3.16.0";
+
+  src = fetchFromGitHub {
+    owner = "hfst";
+    repo = "hfst";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-2ST0s08Pcp+hTn7rUTgPE1QkH6PPWtiuFezXV3QW0kU=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    bison
+    flex
+    pkg-config
+    swig
+  ];
+  buildInputs = [
+    gettext
+    icu
+  ];
+
+  configureFlags = [ "--enable-all-tools" ];
+
+  meta = with lib; {
+    description = "FST language processing library";
+    homepage = "https://github.com/hfst/hfst";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ lurkki ];
+    # Ofborg build error
+    broken = stdenv.isDarwin;
```

Comment:
```suggestion
    platforms = platforms.unix;
```

Reply:
Doesn't `platforms.unix` include Darwin?

Reply:
Sure, unix = linux + darwin.

Reply:
Why should the `broken = stdenv.isDarwin` be removed then?

Reply:
Darwin build is OK now, with changes I suggested above.